### ISSUE TITLE
Duplicated and deleted 

### ISSUE THREAD

**Issue by arrivedercia** [fake-name/wlnupdates#686]
Hello, @admin I never thought about searching my own serial here but when it showed up it made me happy I have no day how you’ve coded this thing XD seems super complicated. https://www.wlnupdates.com/series-id/97997/unities - this is an old series I dropped on my Clearbattery account could you delete it I may post it on another account in the future. https://www.wlnupdates.com/series-id/115096/maths - **and old duplicates of my current serial &** https://www.wlnupdates.com/series-id/117307/maths-maximum-acceleration-of-the-transcending-homo-sapiens -

https://www.wlnupdates.com/series-id/118376/maths-maximum-acceleration-of-the-transcending-homo-sapiens - most recent of everything written but I’m a messing indecisive chapter poster, is there any way I could edit the duplicated chapters? @admin Thank you for any acknowledgement this post can get, this is the first website I’ve seen of its kind seems genius

**Comment by fake-name** [comment]
OK, I've deleted the series for:

 - https://www.wlnupdates.com/series-id/97997/unities
 - https://www.wlnupdates.com/series-id/115096/maths
 - https://www.wlnupdates.com/series-id/117307/maths-maximum-acceleration-of-the-transcending-homo-sapiens

> but I’m a messing indecisive chapter poster, is there any way I could edit the duplicated chapters?

I'm not totally sure what you're trying to say (what is an "indecisive chapter poster"?), but the site is basically a wiki. If you create an account you can edit everything. 

If you have chapters that are missing the issue is /probably/ the chapter extractor not understanding a specific chapter. See the [note about how chapter extractor works](https://www.wlnupdates.com/help#auto-release-requirements). 

Basically, if your series is primarily releases that have chapter numbers, the extractor assumes chapters /without/ numbers are announcements/miscellaneous/etc, and doesn't include them. This was generally because the underlying assumption behind the reading list system is that chapters are monotonic and loosely sequential, and fails if this is not true..

**Comment by arrivedercia** [comment]
My grammar is XD bad what I meant by 

> I’m a messing indecisive chapter poster
Was a messy chapter poster so I might post 10,000 words for a chapter then divide it up into two, with different titles, on the page for my serial I can see that, when I try to delete the duplicates it says I need moderator privileges, only now have I start to be consistent with the naming

**Comment by arrivedercia** [comment]
Sorry for overcomplicating it xD 
https://www.wlnupdates.com/series-id/118376/maths-maximum-acceleration-of-the-transcending-homo-sapiens - basically could you delete this one as well changed, thank you for all the help

**Comment by fake-name** [comment]
Somehow I missed this. Sorry!

Deleted.

**Comment by arrivedercia** [comment]
https://www.wlnupdates.com/series-id/137473/call-of-tardigrade-takes-27-chapters-to-become-an-isekai
https://www.wlnupdates.com/series-id/128497/call-of-tardigrade
https://www.wlnupdates.com/series-id/137392/call-of-tardigrade-revised-amp-continued
https://www.wlnupdates.com/series-id/137551/call-of-tardigrade-an-isekai-with-extra-steps
https://www.wlnupdates.com/series-id/136992/call-of-tardigrade-a-progression-sci-fi-fantasy
@ admin found more duplicates of an old series due to name changes, could you delete these as well, thank you!
